Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -252,7 +252,9 @@ def skip_ineligible_formulae(formula_or_cask)
   end
 
   sig {
-    params(formula_or_cask: T.any(Formula, Cask::Cask)).returns(T.any(Version, String))
+    params(
+      formula_or_cask: T.any(Formula, Cask::Cask),
+    ).returns([T.any(Version, String), T.nilable(T.any(Version, String))])
```

Comment:
Feels like it'd be nicer to keep this as `T.any(Version, String)` and use a null version or empty string?

Reply:
Sure. I was originally thinking of passing both versions (unthrottled, throttled) to print output but would require some extra work.

Right now, I am not using both so it makes sense to collapse it back and just return one of them.